### PR TITLE
Rename repo + replace logo with ACCESS Atmosphere WG logo

### PR DESCRIPTION
**Pull request by heidinett**
Fixes Issue #7 - W21C logo replaced with that of ACCESS Atmosphere WG.

### CODE REVIEW

**Comment by navidcy** [comment]
@angus-g why we don't see the docs previews here? Is it because we transferred the repo?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+<div align="center">
+ <img src="docs/assets/ACCESS_ATMOSPHERE_icon.png" height="300" display="block">   
+</div>
 
 <a href="https://atmospheric-cookbook.readthedocs.io/en/latest/">
     <img alt="latest docs" src="https://img.shields.io/badge/docs-latest-blue.svg">
 </a>
 
-# Atmospheric Cookbook
+# Atmosphere Cookbook
```
Comment (navidcy) [inline README.md:9]: should we rename the repo?

"Atmospheric" characterises the cookbook... that was my thinking

Comment (heidinett) [reply README.md:9]: Yes please @navidcy, that would be great if you could change it to "atmosphere-cookbook" instead of "atmospheric-cookbook"

**Comment by angus-g** [comment]
> @angus-g why we don't see the docs previews here? Is it because we transferred the repo?

I see the preview, unless you're referring to something else?
image

File: README.md (hunk 1 above)
Comment (navidcy) [reply README.md:9]: I can do that, but why not "atmospheric cookbook"? I think atmospheric is more correct grammatically but I'm also not a native English speaker...

**Comment by navidcy** [comment]
> > @angus-g why we don't see the docs previews here? Is it because we transferred the repo?
> 
> I see the preview, unless you're referring to something else? image

I changed the repo settings in the readthedocs and it worked!

**Comment by navidcy** [comment]
Let's converge on the name and then ensure that we change it everywhere (e.g. in the docs also).

**Review by navidcy — CHANGES_REQUESTED** [review]
Let's converge on the name and then ensure that we change it everywhere (e.g. in the docs also).

File: README.md (hunk 1 above)
Comment (heidinett) [reply README.md:9]: I prefer "Atmosphere Cookbook" as it takes on the name of the Atmosphere community, I guess similar to the Cosima Cookbook for the Cosima community.

**Comment by heidinett** [comment]
@navidcy - do we now have consensus on the name "atmosphere cookbook"? 
Would be great to merge this PR before the ACCESS workshop next week.

File: README.md (hunk 1 above)
Comment (navidcy) [reply README.md:9]: Yeah let's do that if you prefer.

Comment (navidcy) [reply README.md:9]: We should change it in the docs and everywhere.

**Comment by heidinett** [comment]
great. Thanks @navidcy!

**Comment by navidcy** [comment]
There are a bunch of "atmospheric" at https://atmospheric-cookbook--9.org.readthedocs.build/en/9/

**Comment by heidinett** [comment]
Would you prefer that I open a new issue to replace "atmospheric" with "atmosphere" in Readthedocs, or address it in this issue?

**Comment by navidcy** [comment]
Let's address it here? Whatever suits you.

**Review by navidcy — APPROVED** [review]
lgtm!

feel free to merge @heidinett when you deal with "atmospheric" -> "atmosphere"

**Comment by heidinett** [comment]
@navidcy, I have replaced references to "atmospheric" with "atmosphere" and created a COPYRIGHT.txt file.

**Pull request merged**